css/base/form.css

```diff
@@ -0,0 +1,69 @@
+::placeholder, ::-ms-input-placeholder, :-ms-input-placeholder{
+  color: var(--placeholder);
+}
+
+.form-control, .form-control:focus{
+  width:324px;
+  height:45px;
+  border-radius: 73px;
+  background-color: var(--tph-tertiary);
+  color:#B9C8D3;
+  font-size:18px;
+  border:none;
+  margin-bottom:22px;
+  padding-left: 26px;
+  padding-right: 26px;
+  box-shadow:none;
+}
+
+.form-password{
+  margin-bottom:22px;
+  display:flex;
+  background-color: var(--tph-tertiary);
+  border-radius: 73px;
+  width:324px;
+  height:45px;
+  padding:0;
+  align-items: center;
+}
+
+.form-password > input{
+  margin-top:22px;
+}
+
+.password, .password:focus{
+  width:281px;
+}
+
+.form-password > i{
+  padding-right:19px;
+  color:var(--tph-text-secondary);
+  font-size:19px;
+}
+
+.form-password > i:hover{
+  cursor:pointer;
+  color:#B9C8D3;
+}
+
+.btn-primary{
+  width:316px !important;
+  height:48px;
+  background-color: var(--tph-text-highlight);
+  text-transform:uppercase;
+  font-size: 21px;
+  border-radius:6px;
```

Comment:
Okay this is honestly super confusing.

There are so many border-radius being implemented in the prototype. **Please** settle for **standard** border-radius for: buttons, content (divs/containers/etc), and anything else that require border-radiuses, and implement them accordingly in the prototype.